library.wdl

```diff
@@ -68,18 +81,11 @@ workflow Library {
             refRefflat = refflatFile
     }
 
-    call preprocess.GatkPreprocess as preprocessing {
-            input:
-                bamFile = markDuplicates.outputBam,
-                basePath = outputDir + "/" + sampleId + "-" + libraryId + ".markdup.bqsr",
-                outputRecalibratedBam = true,
-                splitSplicedReads = true,
-                dbsnpVCF = dbsnp,
-                reference = reference
-    }
+
+
 
     output {
-        IndexedBamFile bamFile = markDuplicates.outputBam
-        IndexedBamFile preprocessBamFile = select_first([preprocessing.outputBamFile])
+        IndexedBamFile bamFile = select_first([markDuplicates.outputBam, starAlignment.bamFile])
```

Comment:
This `select_first` isn't needed anymore since MarkDuplicates is always run again.